book/security.rst

```diff
@@ -0,0 +1,660 @@
+.. index::
+    single: Security
+
+Security
+========
+
+Introduction
+------------
+
+The `OroSecurityBundle`_ sits on top of the Symfony security layer to reach protect your resources.
+This means that each user of your application is granted access to a particular subset of your
+company's resources. Coincidentally, they have to be prevented from accessing resources, access was
+not granted on to them.
+
+Access Control Lists
+--------------------
+
+Access Control Lists are an essential part of the `Symfony Security component`_. They are leveraged
+by the OroSecurityBundle to fulfill the requirements of companies in the business context.
+
+Access Levels
+~~~~~~~~~~~~~
+
+Access can be granted to a user for a certain resource on the several levels. The lowest level is
+the *User* level. Being at this level means that a user can only access resources that have been
+assigned to them or that have been shared with them. At the other end of the hierarchy is the
```

Comment:
For now, we does not have sharing functionality. So I think, it is not good to say about this now.